src/app/phast/phast-report/report-graphs/report-graphs.component.ts

```diff
@@ -76,16 +84,33 @@ export class ReportGraphsComponent implements OnInit {
 
   addChartData(phast: PHAST, name: string, isValid: PhastValid,  modification?: Modification) {
     let valuesAndLabels: Array<{ value: number, label: string }> = this.getValuesAndLabels(phast);
+    let deliverValuesLabels: Array<{ value: number, label: string }> = this.getDeliverValuesAndLabels(phast);
     this.allChartData.push({
       name: name,
       valuesAndLabels: valuesAndLabels,
       barChartLabels: valuesAndLabels.map(valueItem => {return valueItem.label}),
       barChartValues: valuesAndLabels.map(valueItem => {return valueItem.value}),
       modification: modification,
-      valid: isValid
+      valid: isValid,
+      deliverValuesLabels: deliverValuesLabels
     })
   }
 
+  getDeliverValuesAndLabels(phast: PHAST): Array<{ value: number, label: string }> {
+    let results: PhastResults = this.phastResultsService.getResults(phast, this.settings);
```

Comment:
in here we are calling `getResults()`, it also calls that function in the `getValuesAndLabels()` function. both called by `addChartData` 

call `getResults` in the `addChartData` function and pass the results into the other two functions please